app/src/main/java/org/simple/clinic/newentry/PatientEntryEffectHandler.kt

```diff
@@ -123,8 +123,8 @@ class PatientEntryEffectHandler(
 
   private fun hidePhoneLengthErrors() {
     with(validationActions) {
-      showLengthTooLongPhoneNumberError(false)
-      showLengthTooShortPhoneNumberError(false)
+      showLengthTooLongPhoneNumberError(false, 0)
+      showLengthTooShortPhoneNumberError(false, 0)
```

Comment:
Ah, looks like this method is responsible for both hiding *and* showing the error, which now requires us to pass in a parameter which is useless when hiding the error message. A classic violation of single responsibility principle.

We'll have to schedule some work to fix this at some point.

Reply:
Agreed. Hiding errors should have had a different method. I can work on it whenever we do schedule.